### ISSUE TITLE
App Issue on Big Sur ( MacBook Pro 13 )

### ISSUE THREAD

**Issue by MichaelCH1703** [horosproject/horos#607]
MacBook Pro 13 (2019) with retina display and Touch Bar
horos 4.0.0 
Lately I upgraded my MacBook Pro 13 to Big Sur ( I had latest Catalina ) and the main Hub When I open a Scan, shrinks ( Like on the attached screenshoot. The same situation happens on macbook air also with Big Sur upgrade
<img width="1440" alt="Zrzut ekranu 2020-11-16 o 17 44 15" src="https://user-images.githubusercontent.com/74613502/99415280-ed684980-28f7-11eb-8040-b790fd52cf45.png">
Can you guys fix it quickly? or is there any problems with my MacBook configuration?

**Comment by techmdmac** [comment]
Me too in iMac Pro 27"

**Comment by techmdmac** [comment]
addendum : the toolbar that appears deformed is that of MichaelCH1703 (the viewer) but not the MPR or 3D VR.
Also when 3D VR presets are clicked, horos freezes and you have to force quit.

**Comment by techmdmac** [comment]
More: scroll a serie of images in the viewer is unusually slow

**Comment by baccheschi** [comment]
I've been using Horas 4.0.0 RC4 on all Big Sur Public Betas, and the "deformed" toolbar has been present since the first one. Didn't post it because I figured It would be resolved until alfa. I'm still on macOS Big Sur 11.0.1 RC2. But it seems you guys are running 11.0.1 Alpha already and the toolbar is still like this. the bummer is the thick slab that have no other menu alternative but the toolbar. The rest is usable through right-click menus and menu bar. Any solving ahead?

**Comment by maxgmer** [comment]
Any upcoming fixes for this one? Having same issue.

**Comment by fernanpop** [comment]
I have the same problem in MacBook Pro 13 (2017)
And more, scroll a serie of images in the viewer is unusually slow

**Comment by akluiber** [comment]
Same issue here. MacBook Pro (15-inch, 2017). Built-in and external displays.

**Comment by baccheschi** [comment]
> I've been using Horas 4.0.0 RC4 on all Big Sur Public Betas, and the "deformed" toolbar has been present since the first one. Didn't post it because I figured It would be resolved until alfa. I'm still on macOS Big Sur 11.0.1 RC2. But it seems you guys are running 11.0.1 Alpha already and the toolbar is still like this. the bummer is the thick slab that have no other menu alternative but the toolbar. The rest is usable through right-click menus and menu bar. Any solving ahead?

Also forgot to mention the oddly slow scroll, the "not deformed" toolbar on 3D MPR Window and forgot to mention my system, a Mid-2015 MacBook Pro 15 "Core i7" 2.8GHz. Not as new as most of yours, but w/ 16 GB 1600MHz DDR3L SDRAM, 1 TB PCIe-SSD and 2 GB of dedicated VRAM (AMD Radeon R9 M370X).

**Comment by moliveira-fuze** [comment]
The issue I am seeing is similar to and may be related to the one described in this thread. In my case the problem is that I am not able to access the drop down menu that should be part of  "Change the mouse button function". Because of that I cannot enable Dynamic Angle, which is a very important tool to perform my work.
This is blocking my work. Could you please take a look and help me out to fix this issue?

device: macBook Pro 2015, Retina 13 polegadas
os: macOS Big Sur 11.0.1
horos: Version 3.3.6

Actual Response
<img width="1276" alt="Missing DropDown Big Sur" src="https://user-images.githubusercontent.com/50417923/100537289-17d2d480-321f-11eb-8091-025100d48618.png">

Expected Response
<img width="714" alt="Before BigSur" src="https://user-images.githubusercontent.com/50417923/100537321-62545100-321f-11eb-94c1-86f7cbbb8b09.png">

**Comment by aguadamuz1982** [comment]
MacBook pro 2016
I am suffering with this problem with Horos, I have the same issues. But no other DICOM viewer for Mac than Osirix, and is really really expensive. We are not all american radiologists. Horos is used around the globe.

**Comment by baccheschi** [comment]
I've just realized I stopped having this issue on macOS Big Sur 11.1 Beta (20C5048k). I always use my MacBookPro11,5 on a 4K external display when I'm at home. But I have always used it on a scaled resolution of 2560 x 1440, which was making Big Sur run really slow, whenever running a virtual machine on VMWare Fusion (now on Fusion Player 12.1 version) and "stealing" 8 GB for the virtual machine. Then I read somewhere that using the external display on native resolution (3840 x 2160) would compromise less of Big Sur performance when running Horos and tiling multiple windows. Maybe for not having to scale display? I'm not sure. But fact is, and this is quite subjective, that it actually happened for me. It was then that I realized that the toolbar issue was gone. And it doesn't happen on any resolution anymore, neither on external display, nor on the MacBook display. But since I upgraded to Beta 11.1 simultaneously, I'm not sure what did the trick. Maybe It was just the beta version. But the not scaled display performance improvement on native 4K resolution it's true. The bummer is that you have to upscale every app possible on macOS and system text get really tiny on a 27" 4K monitor. What's easier on Windows 10 guest system when running the virtual machine. I've always took for granted the macOS "using a scaled resolution may affect performance" message. Not anymore.

**Comment by fvpolpeta** [comment]
Please, if possible, help us to confirm that version 4.0.0 RC5 is no longer
presenting this issue.

It's available for download at:

https://horosproject.org/download/

Thanks

Fauze

**Comment by baccheschi** [comment]
Nope. Still not happening. Toolbar is normal in Horos 4.0.0 RC5, running on macOS Big Sur 11.1 Beta (20C5048k), MacbookPro 11,5 with every scaled resolutions and default.

**Comment by techmdmac** [comment]
Toolbar looks normal in RC5.

**Comment by akluiber** [comment]
Yep, RC5 has fixed the issue for me on Big Sur 11.0.1

**Comment by techmdmac** [comment]
Ja, ja, ja New version of Horos for BUG Sur, je, je... Very ingenious !!!

**Comment by techmdmac** [comment]
In my opinion Horos 4 RC5 works generally well. The only problem I see that persists (and annoying) is the slow scrolling in viewer mode. The rest of issues, for me are solved.

**Comment by fernanpop** [comment]
Is the same for me. Srcoll is slow in viewer mode and ct coronal and
sagital reformat.

Em sáb, 5 de dez de 2020 11:01, techmdmac <notifications@github.com>
escreveu:

**Comment by shelars** [comment]
I am using Horos 4.0.0 RC4 on MacBook pro (late 2017) with Catalina and it works flawlessly. My conclusion is that version' s RC5 major problem is very slow scrolling. So I won' t upgrade to Big Sur until a newer version of Horos is released. My only question is that. Do the MacBooks with this problem use Intel graphic card? My card is **Intel Iris Plus Graphics 640 1536 MB**

**Comment by tomtart1962** [comment]
Using Big sur and version 4.0 RC5 the Query command dose not work anymore
Do you have the same problem?

**Comment by techmdmac** [comment]
No, I have several problems but not that you have: 3D VR extremly displaced from the center when you rotate or clip the image, slow scroll, etc.
